ci/run_ci.sh

```diff
@@ -46,7 +46,7 @@ do
   start_time=`date`
   my_dir=${dir::-1}
   figlet "CI test for "$my_dir
-  if $my_dir/ci_test.sh
+  if [ -f $my_dir/ci_test.sh ] && $my_dir/ci_test.sh
```

Comment:
just this change alone will still cause failures in CI as the rc is set in else part. Unless im reading this wron

Reply:
You're right! I pushed some changes fixing this